packages/pods-core/src/v2/ConfigFileUtils.ts

```diff
@@ -67,4 +73,44 @@ export class ConfigFileUtils {
     }
     return fPath;
   }
+
+  static createExportConfig(opts: { required: string[]; properties: any }) {
```

Comment:
Let's try to add some typing to this.  I think if we add a generic param T, and have something like 

`opts: { required: (keyof T)[]; properties: T}`

_Exact typing syntax needs to be fixed_